dashboard/templates/base.html

```diff
@@ -28,6 +28,34 @@
         }
     </style>
     {{ myla_globals|json_script:"myla_globals" }}
+    <script>
+        var googleAnalyticsId = JSON.parse(document.getElementById('myla_globals').textContent).google_analytics_id;
+        window.umConsentManager = {
```

Comment:
I think this script can be minimize to just be 
```
<script>
    window.umConsentManager = {
        googleAnalyticsID: "{{ myla_globals.google_analytics_id }}", // e.g. G-XXXXXXXXXX
        googleAnalyticsCustom: {
            streamConfig: {
                cookie_flags: "SameSite=None; Secure"
            }
        }
    };
</script>
```

- myla_globals should be in the Django template context so can be accessed directly
- All of the other code appears to just be the default

Reply:
Also, add in the option to this for 

`    externalLinkBlank: true
`